e2e/wdio/core/tests/message-toast.e2e-spec.ts

```diff
@@ -0,0 +1,83 @@
+import { MessageToastPo } from '../pages/message-toast.po';
+import {
+    waitForElDisplayed,
+    refreshPage,
+    click,
+    isElementDisplayed,
+    pause,
+    mouseHoverElement,
+    doesItExist,
+    scrollIntoView,
+    getElementArrayLength,
+} from '../../driver/wdio';
+
+describe('Textarea component test', function () {
+    const messageToastPage = new MessageToastPo();
+    const {
+        openMessageButton,
+        hideAllButton,
+        message
+    } = messageToastPage;
+
+    beforeAll(() => {
+        messageToastPage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForElDisplayed(messageToastPage.title);
+    }, 2);
+
+    it('should check message from component', () => {
```

Comment:
should check what? please make these more descriptive

'should check the component example message popup appears' or ' should check component example message appears on click'

same for the other test cases below